src/scss/components/_menu.scss

```diff
@@ -2,9 +2,21 @@
   * @parent gaiden_css.components
   * @stylesheet gaiden_css.components.menu.css Menu
   *
-  * @demo demo/gaiden-css/components/menu/demo.html
 **/
 
+/**
+  * @styles menu_over Over
+  *
+  * @demo demo/gaiden-css/components/menu/default/demo.html
+**/
+
+/**
+  * @styles menu_neutral Neutral
+  *
+  * @demo demo/gaiden-css/components/menu/neutral/demo.html
+**/
+
+
 .menu {
   @include media(small) {
```

Comment:
Um outro ponto, esse `flex-direction: row;` que está sendo incluido no breakpoint antes mesmo das propriedades default da classe não estão sendo anuladas pelo efeito cascata?

Logo depois é carregado um `flex-direction: column;`

Reply:
@williamcosta this will be overrided only when breakpoint was bigger than small. 

Are you talking about the mixin be called at bottom of seletor?

Reply:
@williamcosta It sounds weird but it does not override 🤔 . I will move the mixin after the default props. 👍

Reply:
Yes @eduardojmatos I think it's kind of standard to put the breakpoints after the default properties.

Nice @iondrimba

Reply:
@williamcosta our Sass Lint requires to all mixins calls be declared at top of seletor (cc @iondrimba)

Reply:
And this will not override another CSS because is a media query wrapper

Reply:
*be override

Reply:
I understood @eduardojmatos  I am more accustomed to using the breakpoint declarations after the standard declarations of the class, but if this works and that is what our SASS Lint asks for, that's fine.

Reply:
Rolling back to top of the declaration 👍